test/processing/social-media-api-test.coffee

```diff
@@ -0,0 +1,171 @@
+  'lmuexp'
+  'LALoyolan'
+  'LoyolaLawSchool'
+  'LoyolaMarymount'
+  'MEforLMU'
+  'LMUMARCOMM'
+  'lmumbaprogram'
+  'lmunewsroom'
+  'LMUCareers'
+  'LMU_OISS'
+  'LMUPlaceCorps'
+  'roarstudios'
+  'lmusoe'
+  'LMUDoctoral'
+  'LMUsftv'
+  'seaverlmu'
+  'lmuhousing'
+  'studyabroadlmu'
+  'LMUTFALA'
+  'LMUTFANorCal'
+  'LMUTower'
+  'tsehai'
+  'LMUAdmission'
+]
+
+fbScreenNames = [
+  "actilmu"
+  "groups/lmuaaaa"
+  "lmualumni"
+  "lmustararhs"
+  "aspalmu"
+  "LMUWomensVolleyball"
+  "LMULionTRaXC"
+  "lmumensbasketball"
+  "GolfLMU"
+  "LMUMTennis"
+  "LMURowing"
+  "LoyolaMarymountUniversitySoftball"
+  "LMUSwimming"
+  "LMUWomensBasketball"
+  "LMUWomensSoccer"
+  "LMUWomensTennis"
+  "LmuWomensWaterPolo"
+  "LMULions"
+  "BiologyLMU"
+  "LmuBurnsRecreationCenter"
+  "lmucampusmin"
+  "LMUCAST"
+  "groups/39893463773"
+  "lmucrs"
+  "LMUCSA"
+  "LMUCSLA"
+  "lmucures"
+  "lmucba"
+  "lmucommunityrelations"
+  "LMUCS"
+  "LMUConferences"
+  "LMUdanceprogram"
+  "lmuhospitality"
+  "LMU-English-Department-280903465269783"
+  "LMUEIS"
+  "LMUEMBA"
+  "LMUExtension"
+  "LMUFacultySenate"
+  "lmufos"
+  "LMUGradAdmission"
+  "lmugreeklife"
+  "lmulibrary"
+  "LMUHistoryDepartment"
+  "leadershiplmu"
+  "LMUInternationalOutreach"
+  "irishstudieslmu"
+  "KXLU889"
+  "Laband-Art-Gallery-255909641096056"
+  "LmuLatinoAlumniAssociation"
+  "lmulionparents"
+  "lmucares"
+  "lmusoe"
+  "LosAngelesLoyolan"
+  "LoyolaLawSchool"
+  "LLSAlumniNetwork"
+  "lmula"
+  "LMU-M-School-497182070316821"
+  "ManeEntertainmentLMU"
+  "Marymount-Institute-Press-180302548725093"
+  "LMUMBAProgram"
+  "lmucareers"
+  "oisslmu"
+  "lmupols"
+  "lmuschoolpsychology"
+  "roarstudiosla"
+  "LMUDoctoral"
+  "LMUSFTV"
+  "LMUCenterforStudentSuccess"
+  "lmuseaver"
+  "LMUStudentHousing"
+  "LMUTFANorCal"
+  "tsehaipublishers"
+  "lmuadmission"
+  "lmuyoga"
+  "lmuyogastudies"
+]
+
+describe 'Making social media requests', ->
+  describe 'when making Facebook API calls', ->
+    for name in fbScreenNames
+      do (name) ->
+        it "We get a 200 from #{name}", (done) ->
+          requestURL = "#{serverName}#{fbPostURL}#{name}"
+          request requestURL, (err, res, body) ->
+            expect(res.statusCode).to.eql 200
+            expect((JSON.parse body).data).to.not.eql undefined
+            done()
+
+  describe 'when making Twitter API calls for a single user', ->
+    for name in twitterScreenNames
+      do (name) ->
+        it "We get a 200 from #{name}", (done) ->
+          requestURL = "#{serverName}#{twitterUsersURL}#{name}"
+          request requestURL, (err, res, body) ->
+            expect(res.statusCode).to.eql 200
+            done()
+
+    describe 'when making Twitter API calls for a list', ->
+      it 'We get a 200', (done) ->
+        list_id = 222479944
+        requestURL = "#{serverName}#{twitterListURL}#{listName}"
+        request requestURL, (err, res, body) ->
+          expect(res.statusCode).to.eql 200
+          done()
```

Comment:
I think we need to name this `*_spec.coffee` so it will be discoverable by the test script.